Mailbox.Tests/DataLoaderTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Mailbox.Tests
+{
+    [TestClass]
+    public class DataLoaderTests
+    {
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void DataLoader_Load_StreamIsNull_ThrowNullExeption()
```

Comment:
No need to have "DataLoader" in the name `DataLoader_Load_StreamIsNull_ThrowNullExeption()` since it is in the `DataLoaderTests` class. Here are [Best Practices](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices) for writing unit tests if your curious, really helped me out earlier.
```suggestion
        public void Load_StreamIsNull_ThrowNullExeption()
```